docs/operators.md

```diff
@@ -36,7 +40,7 @@ While the SDK is geared toward this type of design, you can still put all of you
 if you either need to have calls be completely synchronous, or the business logic is very simple.
 
 **Forthcoming:** the SDK codegen utility offers the ability to create a simple CRUDL backend plugin API 
-and an operator template given only one or more Schemas defined in CUE (see [Schema Management](schema_management.md)). 
+and an operator template given only one or more Schemas defined in CUE (see [Kinds documentation](custom-kinds/README.md)).
```

Comment:
This is pointing to docs created in https://github.com/grafana/grafana-app-sdk/pull/240/files

Reply:
This is actually no longer forthcoming, `grafana-app-sdk project component add backend` does this, though the CRUDL plugin backend API is also something we want to slowly move away from in favor of direct API server access.

Reply:
LGTM!